spec/controllers/polls_controller_spec.rb

```diff
@@ -19,6 +23,32 @@
     end
   end
 
+  describe 'GET #ready_index' do
+    before(:each) do
+      FactoryGirl.create(:valid_poll)
+      FactoryGirl.create(:valid_poll)
+      FactoryGirl.create(:valid_poll, :with_ready_status)
+    end
+
+    it 'should succesful get ready index' do
+      expect(get :ready_index).to be_succes
+    end
+
+    it 'should render ready index template' do
+      expect(get :ready_index).to render_template(:ready_index)
+    end
+
+    it 'should render ready polls' do
+      get :ready_index
+      expect(assigns(:polls)).to eq(Poll.ready)
```

Comment:
`eq` is really fragile here, you should use `match_array` in such cases. It is order-independent, so `[poll1, poll2]==[poll2, poll1]` will fail, but match_array won't — and DB is not guaranteeing the order of rows in output unless you make an explicit `order` call.